node/Makefile

```diff
@@ -294,15 +276,18 @@ $(NODE_CONTAINER_CREATED): register ./Dockerfile.$(ARCH) $(NODE_CONTAINER_FILES)
 	touch $@
 
 # download BIRD source to include in image.
-$(BIRD_SOURCE): ../go.mod
+$(BIRD_SOURCE): .bird-source.created
+.bird-source.created:
 	mkdir -p filesystem/included-source/
 	wget -O $@ https://github.com/projectcalico/bird/tarball/$(BIRD_VERSION)
```

Comment:
won't `-O $@` write out the tarball as a file named `.bird-source.created` ?  Seems like you should be able to keep the name of the target being the name of the tarball.

I recommend slipping in an `rm filesystem/included-source/bird*` to clean up any stale tarballs form earlier runs.